fusion-cli/build/jest/base-jest-config.js

```diff
@@ -9,14 +9,19 @@
 /* eslint-env node */
 const fs = require('fs');
 const {dirname} = require('path');
+const loadFusionRC = require('../load-fusionrc.js');
 
 const rootDir = dirname(fs.realpathSync(`${process.cwd()}/package.json`));
 
+const {jest: jestConfig, jsExtPattern} = loadFusionRC(rootDir);
+const testFileExt = jsExtPattern ? jsExtPattern.source : '\\.js$';
+
+const TRANSFORM_IGNORE_PATTERNS = ['/node_modules/(?!(fusion-cli.*build))'];
 const matchField = process.env.TEST_REGEX ? 'testRegex' : 'testMatch';
 const matchValue = process.env.TEST_FOLDER
-  ? [`**/${process.env.TEST_FOLDER || '__tests__'}/**/*.js`]
+  ? [`**/${process.env.TEST_FOLDER || '__tests__'}/**/*${testFileExt}`]
   : process.env.TEST_REGEX ||
-    (process.env.TEST_MATCH || '**/__tests__/**/*.js').split(',');
+    (process.env.TEST_MATCH || `**/__tests__/**/*${testFileExt}`).split(',');
```

Comment:
I'm not sure using `testFileExt` will work within a glob here since `jsExtPattern` is regex

Reply:
Yep you are right, I'll address this